resources/css/directory.css

```diff
@@ -226,8 +233,10 @@ ul.directory-list.filterable {
   }
 }
 .directory ul.directory-list li .image-container {
-  width: 6.5em;
-  height: 9.1em;
+  width: 18em;
+  height: 18em;
+  /* width: 6.5em;
```

Comment:
Remove the commented out width and height. Those are good for development, but not for production